raw_macaroons.go

```diff
@@ -0,0 +1,171 @@
+package lndclient
+
+import (
+	"encoding/base64"
+	"encoding/hex"
+	"fmt"
+)
+
+// RawMacaroons contains raw []byte macaroons
+// for later hex encoding and use with LND.
+type RawMacaroons struct {
+	AdminMac     []byte
+	InvoiceMac   []byte
+	ChainMac     []byte
+	SignerMac    []byte
+	WalletKitMac []byte
+	RouterMac    []byte
+	ReadOnlyMac  []byte
+}
+
+// Returns a RawMacaroons object populated with raw []byte macaroons.
+// Best used if loading macaroons directly from files, or if reading from
+// Kubernetes secrets containing raw byte data (for example)
+func NewRawMacaroons(adminMac []byte, invoiceMac []byte,
+	chainMac []byte, signerMac []byte, walletKitMac []byte,
+	routerMac []byte, readOnlyMac []byte) *RawMacaroons {
```

Comment:
I think this method can be removed. There's already quite few methods exposed for creating a new struct, and I think it's a good thing to expose as few public methods as possible from a package. Could possibly remove some of the other public methods as well?